src/QueryBuilder.php

```diff
@@ -99,7 +99,7 @@ public function setParams(array $params): self
     public function build(): array
     {
         $query = $this->params;
-        if (!isset($query['body'])) {
+        if (isset($query['body']) === false) {
```

Comment:
vyssie pouzivas yoda comparison. neviem ako to maju v projekte, ale pre konzistentnost by som to spravil aj tu (false === isset..)

Reply:
toto spravil automaticky ich phpstan